packages/suite-native-atoms/src/VStack.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactNode } from 'react';
+
+import { NativeStyleObject, prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { NativeSpacing, nativeSpacings } from '@trezor/theme';
+
+import { Box } from './Box';
+
+type VStackProps = {
+    children: ReactNode;
+    style?: NativeStyleObject;
+    spacing?: NativeSpacing;
+};
+
+type SpacerStyleProps = {
+    isLastChild: boolean;
+    spacing: NativeSpacing;
+};
+
+const spacerStyle = prepareNativeStyle<SpacerStyleProps>((_, { isLastChild, spacing }) => ({
+    extend: {
+        condition: !isLastChild,
+        style: {
+            marginBottom: spacing,
+        },
+    },
+}));
+
+export const VStack = ({ children, style, spacing = nativeSpacings.small }: VStackProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <Box style={style}>
+            {React.Children?.map(children, (child, index) => (
+                <>
+                    <Box>{child}</Box>
+                    <Box
```

Comment:
why empty box? why don't apply this style directly to previous Box?

Reply:
Ah, that was wrong ofc. Fixed in [bac2f35](https://github.com/trezor/trezor-suite/pull/5583/commits/bac2f35708091d37f278d3267c42c93a0c26edf9)